HHS/Head-Start-TTADP
# [TTAHUB-1517] Fix visual bug with site alerts on recipient record pages

## Description of change
This PR fixes a visual bug with the site alerts. The bug appears only the recipient record pages that have the sub navigation at the top.

Before:
![Screenshot 2023-02-22 at 12 39 57 PM](https://user-images.githubusercontent.com/3288586/220710694-9fe2c757-ca07-4fa6-b188-75cd53f63d0c.png)

After:
![Screenshot 2023-02-22 at 12 39 28 PM](https://user-images.githubusercontent.com/3288586/220710727-f77652b8-d54f-4f9d-80f3-387bc93931e3.png)

This PR also adds a feature flag called "rttapa_form" to the database.

## How to test
Staging has these alerts. Visit the recipient record page and confirm that there are no issues with the alerts overlapping the navigation.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1517


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
